systolic/test/Cgra3x3MemRightAndBottomRTL_matmul_2x2_test.py

```diff
@@ -281,17 +281,17 @@ def test_CGRA_systolic(cmdline_opts):
                     FuOutType (0), FuOutType (0), FuOutType (0), FuOutType (0)])],
 
       # On tile 2 ([0, 2]).
-      [
-       CtrlPktType(0,  2,  0,    0,  CMD_CONFIG, 0,   OPT_NAH, b1(0), pick_register,
-                   [TileInType(0), TileInType(0), TileInType(0), TileInType(0),
-                    TileInType(0), TileInType(0), TileInType(0), TileInType(0)],
-                   [FuOutType (0), FuOutType (0), FuOutType (0), FuOutType (0),
-                    FuOutType (0), FuOutType (0), FuOutType (0), FuOutType (0)]),
-       CtrlPktType(0,  2,  0,    0,  CMD_LAUNCH, 0,   OPT_NAH, b1(0),    pick_register,
-                   [TileInType(0), TileInType(0), TileInType(0), TileInType(0),
-                    TileInType(0), TileInType(0), TileInType(0), TileInType(0)],
-                   [FuOutType (0), FuOutType (0), FuOutType (0), FuOutType (0),
-                    FuOutType (0), FuOutType (0), FuOutType (0), FuOutType (0)])],
+      # [
```

Comment:
Plz add comment here to indicate tile 2 doesn't need to do anything. And create a folder, i.e., `VectorCGRA -> doc -> figures`, upload the following figure, and embed its link in this comment.

![image](https://github.com/user-attachments/assets/09f01b1d-fc17-49f8-9d68-57d0c3925e12)